<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>450</width>
                <height>280</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>过滤行</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
                <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QFrame" name="frame_3">
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <widget class="QFrame" name="frame_4">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                            <layout class="QVBoxLayout" name="verticalLayout">
                                                <property name="spacing">
                                                    <number>0</number>
                                                </property>
                                                <property name="leftMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="topMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="rightMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="bottomMargin">
                                                    <number>0</number>
                                                </property>
                                                <item>
                                                    <widget class="QLabel" name="label">
                                                        <property name="text">
                                                            <string>可以多选</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QListWidget" name="column_names">
                                                        <property name="selectionMode">
                                                            <enum>QAbstractItemView::ExtendedSelection</enum>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QFrame" name="frame_6">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                            <layout class="QVBoxLayout" name="verticalLayout_2">
                                                <item>
                                                    <widget class="QFrame" name="frame_8">
                                                        <property name="frameShape">
                                                            <enum>QFrame::StyledPanel</enum>
                                                        </property>
                                                        <property name="frameShadow">
                                                            <enum>QFrame::Raised</enum>
                                                        </property>
                                                        <layout class="QHBoxLayout" name="horizontalLayout_5">
                                                            <item>
                                                                <widget class="QLabel" name="label_2">
                                                                    <property name="text">
                                                                        <string>查看后，可以删除</string>
                                                                    </property>
                                                                </widget>
                                                            </item>
                                                        </layout>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QFrame" name="frame_7">
                                                        <property name="frameShape">
                                                            <enum>QFrame::StyledPanel</enum>
                                                        </property>
                                                        <property name="frameShadow">
                                                            <enum>QFrame::Raised</enum>
                                                        </property>
                                                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                                                            <property name="leftMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="topMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="rightMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="bottomMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <item>
                                                                <widget class="QCheckBox" name="checkBox_null">
                                                                    <property name="text">
                                                                        <string>含有空值</string>
                                                                    </property>
                                                                </widget>
                                                            </item>
                                                        </layout>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QFrame" name="frame_5">
                                                        <property name="frameShape">
                                                            <enum>QFrame::StyledPanel</enum>
                                                        </property>
                                                        <property name="frameShadow">
                                                            <enum>QFrame::Raised</enum>
                                                        </property>
                                                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                                                            <property name="spacing">
                                                                <number>7</number>
                                                            </property>
                                                            <property name="leftMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="topMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="rightMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="bottomMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <item>
                                                                <widget class="QCheckBox" name="checkBox_tougao">
                                                                    <property name="text">
                                                                        <string>含有投稿</string>
                                                                    </property>
                                                                    <property name="checked">
                                                                        <bool>true</bool>
                                                                    </property>
                                                                </widget>
                                                            </item>
                                                            <item>
                                                                <widget class="QCheckBox" name="checkBox_bianjibu">
                                                                    <property name="text">
                                                                        <string>含有《编辑部》</string>
                                                                    </property>
                                                                    <property name="checked">
                                                                        <bool>true</bool>
                                                                    </property>
                                                                </widget>
                                                            </item>
                                                        </layout>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QFrame" name="frame">
                                                        <property name="frameShape">
                                                            <enum>QFrame::StyledPanel</enum>
                                                        </property>
                                                        <property name="frameShadow">
                                                            <enum>QFrame::Raised</enum>
                                                        </property>
                                                        <layout class="QHBoxLayout" name="horizontalLayout">
                                                            <property name="spacing">
                                                                <number>7</number>
                                                            </property>
                                                            <property name="leftMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="topMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="rightMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="bottomMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <item>
                                                                <widget class="QPushButton" name="btn_ok">
                                                                    <property name="styleSheet">
                                                                        <string notr="true">color: rgb(255, 255, 255);
                                                                            background-color: rgb(28, 177, 245);
                                                                        </string>
                                                                    </property>
                                                                    <property name="text">
                                                                        <string>删除</string>
                                                                    </property>
                                                                </widget>
                                                            </item>
                                                        </layout>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
